.gitignore

```diff
@@ -11,3 +11,4 @@ rust/core/Cargo.lock
 rust/wasm/target/**
 rust/wasm/Cargo.lock
 rust/json-gen-split/target/**
+tools/metadata-cddl-checker/Cargo.lock
```

Comment:
Shouldn't this not be here since this a tool people use as an application not as a library used by other rust programs?

Reply:
currently there's no .lock in that folder, we either should ignore the file in git or generate and add